app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { getTopics } = require("./controllers/app.controller");
+const { handleCustomErrors, handleServerErrors } = require("./errors");
+const app = express();
+app.use(express.json())
+
+app.get('/api/topics', getTopics)
+
+app.use(handleCustomErrors)
```

Comment:
Same as above, there are no custom errors at this point, so this PR shouldn't include handling custom errors.